File: LeetCode/py/905.py
#Link: https://leetcode.com/problems/sort-array-by-parity/

class Solution:
    def sortArrayByParity(self, A: List[int]) -> List[int]:
        return list(filter(lambda x: x%2 == 0, A)) + list(filter(lambda x: x%2 == 1, A))
        

File: LeetCode/cpp/905.cpp
//Link: https://leetcode.com/problems/sort-array-by-parity/

class Solution {
public:
    vector<int> sortArrayByParity(vector<int>& A) {
        vector<int> res;
        for(auto x : A) if(x%2 == 0) res.push_back(x);
        for(auto x : A) if(x%2 == 1) res.push_back(x);
        return res;
    }
};
